### PR TITLE
Fix make_sys_basic_types.py for Fedora 22

### PR DESCRIPTION
**Pull request by mppf**
Continuing #1748. I wanted to check if 'make check' worked on Fedora 22
and ran into errors with this script. That version of the compiler is emitting
#line type directives and whitespace between the relevant numbers for some reason.
Update the script to ignore these lines.

### CODE REVIEW

**Comment by mppf** [comment]
@thomasvandoren can you review this?

File: util/config/make_sys_basic_types.py [hunk 1]
```diff
@@ -127,6 +127,15 @@ def get_sys_c_types(docs=False):
         elif 'FIND_INT_SIZES_START' in line:
             keep = True
 
+        # Ignore lines starting with # since they could be #line
+        # type directives.
+        elif line.startswith("#"):
```
Comment (thomasvandoren) [inline util/config/make_sys_basic_types.py]: Could there be whitespace between the start of the line and the `#`? If so, you might change this to: `line.strip().startswith('#')`.

File: util/config/make_sys_basic_types.py [hunk 2]
```diff
@@ -127,6 +127,15 @@ def get_sys_c_types(docs=False):
...
+            continue
+
+        # Ignore blank lines
+        elif line.isspace() or line == '':
```
Comment (thomasvandoren) [inline util/config/make_sys_basic_types.py]: This collapses to `elif not line.strip():`.

**Comment by thomasvandoren** [comment]
Thanks for fixing this!

**Pull request merged**